src/mixins/bcol-mixin.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Vue } from 'vue-property-decorator'
+import axios from '@/utils/axios-auth'
+
+@Component({})
+export default class BcolMixin extends Vue {
+  async getErrorObj (errCode): Promise<any> {
```

Comment:
Might as well specify what type `errCode` is (and `error` below).

Reply:
done